modules/mono/glue/GodotSharp/GodotSharp/Core/Attributes/ToolAttribute.cs

```diff
@@ -2,6 +2,21 @@
 
 namespace Godot
 {
+    /// <summary>
+    /// Allows the annotated class to execute in the editor.
+    /// </summary>
+    /// <example>
+    /// <code>
+    /// [Tool]
+    /// public partial class MySprite : Sprite2D
+    /// {
+    ///     public override void _Process(double delta)
+    ///     {
+    ///         Rotation += Mathf.Pi * (float)delta;
+    ///     }
+    /// }
+    /// </code>
+    /// </example>
```

Comment:
I'm not sure this example is very useful. Also, how are examples in C# documentation surfaced to users? I don't think VSCode has any way of displaying them and since we don't use tools like DocFX then this is probably not visible for most users.

Reply:
I honestly have no clue how examples get shown in C# documentation; it's a pretty glaring oversight in most systems. Examples are utilized already in the current documentation (albeit rarely), so it's not entirely without precedent. I'm fine with ditching it if it seems superflous, as I don't even fully recall *why* I added this to begin with, but I wouldn't say it's hurting anything either.

Reply:
I'm not saying we shouldn't use examples, but this one in particular doesn't seem that useful. Usually an example shows how to use an API, but in this case the API is just `[Tool]`. There's nothing special about using `[Tool]` compared to any other attribute, the important part is the behavior which can't really be shown with a code example IMO.